CHANGELOG.md

```diff
@@ -30,39 +30,42 @@ Individual unreleased changelog entries can be located at [changelog/unreleased]
 
 - Fixed an inefficiency issue in the Luajit hashing algorithm
  [#13240](https://github.com/Kong/kong/issues/13240)
- 
+ [KAG-4646](https://konghq.atlassian.net/browse/KAG-4646)
```

Comment:
We need to remove these internal tickets from this file.

Reply:
fixed